mne_bids/write.py

```diff
@@ -1543,8 +1589,77 @@ def write_raw_bids(raw, bids_path, events_data=None,
     return bids_path
 
 
-def write_anat(image, bids_path, raw=None, trans=None, landmarks=None,
-               t1w=None, deface=False, overwrite=False, verbose=False):
+def get_anat_landmarks(image, info, trans, fs_subject, fs_subjects_dir=None):
+    """Get landmarks transformed to image voxel coordinates from head space.
+
+    Parameters
+    ----------
+    image : str | pathlib.Path | NibabelImageObject
+        Path to an MRI scan (e.g. T1w) of the subject. Can be in any format
+        readable by nibabel. Can also be a nibabel image object of an
+        MRI scan. Will be written as a .nii.gz file.
+    info : mne.Info
+        The measurement information from an electrophysiology recording of
+        the subject with the anatomical landmarks stored in its
+        :class:`mne.channels.DigMontage`.
+    trans : mne.transforms.Transform | str
+        The transformation matrix from head to MRI coordinates. Can
+        also be a string pointing to a ``.trans`` file containing the
+        transformation matrix. If ``None`` and no ``landmarks`` parameter is
+        passed, no sidecar JSON file will be created.
+    fs_subject : str
+        The subject identifier used for the freesurfer recon-all. If None,
+        defaults to the ``sub`` in ``bids_path``. Must be provided to write
+        the anatomical landmarks if they are not provided in mri voxel space.
+        This is because the head coordinate of a
+        :class:`mne.channels.DigMontage` is aligned using freesurfer surfaces.
```

Comment:
```suggestion
        The subject identifier used for FreeSurfer. If ``None``, defaults to
        the ``subject`` entity in ``bids_path``. Must be provided to write
        the anatomical landmarks if they are not provided in MRI voxel space.
        This is because the head coordinate of a
        :class:`mne.channels.DigMontage` is aligned using FreeSurfer surfaces.
```

Reply:
this needs fixing as there is only an image param here and no bids_path